chat-widget/stories/OmniChannelLiveChatWidget.stories.tsx

```diff
@@ -113,4 +113,242 @@ const liveChatWidgetDefaultWithSurveyProps: ILiveChatWidgetProps = {
     telemetryConfig: undefined
 };
 
-DefaultWithSurvey.args = liveChatWidgetDefaultWithSurveyProps;
\ No newline at end of file
+DefaultWithSurvey.args = liveChatWidgetDefaultWithSurveyProps;
```

Comment:
Please share some screenshots on different components for this custom widget (please link in description), thanks!

Reply:
Updated